### DISCUSSION TITLE
Trying install older vercel and getting 404 from npm

### DISCUSSION THREAD

**Discussion by dmi3y** [vercel/community#3254] [Help] [answered]
### Summary

I'm getting registry error 404 when installing older vercel package (it worked just fine a day before)

Here is command to reproduce
> npx vercel@^28.18.1 --help

<img width="1259" alt="image" src="https://github.com/vercel/community/assets/1277279/bc45c3c0-0839-4e88-9db0-74e63dd58981">
****

Anyone know what's up?

### Steps to Reproduce

Run command
> npx vercel@^28.18.1 --help

UPD:

Noticed similar error when trying to use latest version:

> npx vercel --help

```bash
npm ERR! code E404
npm ERR! 404 Not Found - GET https://registry.npmjs.org/@vercel%2fnext - Not found
npm ERR! 404
npm ERR! 404  '@vercel/next@3.8.8' is not in this registry.
npm ERR! 404
npm ERR! 404 Note that you can also install from a
npm ERR! 404 tarball, folder, http url, or git url.
```

**Comment by swarnava** [comment, marked as answer]
NPM registry having an outage. You can learn more here https://www.vercel-status.com/incidents/3xls7vlwkgh5

**Reply by dmi3y** [reply]
Thanks!
